src/components/Footer.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Footer = () => (
+  <div>
+    <h2>Author's Haven Footer </h2>
+  </div>
+);
+
+
+export default Footer;
```

Comment:
Nice job completing this task, but you need to check your files for EOL's and add them.

Reply:
No EOL present